Bottom: 324f4dc25ece20f0d9655b8ba9b849b41a5150df
Top:    61ad1ee69559a255aad183d770153fc68a1ee621
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 14:33:58 +0000

Refresh of adjust-step-export-to-0-2mm

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a9065ab..a96459f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -60,7 +60,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 
 static GList *object3d_test_objects = NULL;
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 02e9528..8150c46 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 
 HID step_hid;
